src/Exceptions/BadRequest.php

```diff
@@ -7,10 +7,36 @@
 
 class BadRequest extends Exception
 {
+    /**
+     * The dropbox error code supplied in the response
+     *
+     * @var string
+     */
+    protected $dropboxCode = '';
+
     public function __construct(ResponseInterface $response)
     {
         $body = json_decode($response->getBody(), true);
 
+        if ($response->getStatusCode() === 409) {
```

Comment:
Instead of checking the status code, just check if `$body['error']['.tag']` has been set on `$body`.